### ISSUE TITLE
Latest version is not latest on the website

### ISSUE THREAD

**Issue by AlekSi** [containrrr/shoutrrr#293]
https://containrrr.dev/shoutrrr/v0.6/ gives me:

> You're not viewing the latest version. Click here to go to latest.

Where latest leads to 0.5 docs.

<img width="528" alt="CleanShot 2022-11-23 at 18 56 06@2x" src="https://user-images.githubusercontent.com/11512/203577907-80bd41ae-79c3-4f50-abec-bbc50082d0a4.png">

**Comment by piksel** [comment]
Yes, something is wrong with the docs generation. I will look into it. Thanks for reporting!

**Comment by andig** [comment]
This is still an issue. Selecting 0.7 will warn "not viewing latest release".

**Comment by piksel** [comment]
Yeah, I noticed. I will make a manual fix as soon as I get back.

**Comment by piksel** [comment]
Found the missing piece. This needs to be run whenever the minor changes:
```
mike deploy --push --update-aliases 0.7 latest
```

I will keep this open until it has been added to CI, but at least it's fixed in the current docs.
